pkg/platform/runtime/setup/implementations/installer/runtime.go

```diff
@@ -0,0 +1,32 @@
+package installer
```

Comment:
Can we name this "offline" or something. "installer" doesn't convey anything useful that differentiates it from the other implementations.